platform/Span.h

```diff
@@ -49,29 +49,29 @@ class is_convertible
 
 /**
  * Special value for the Extent parameter of Span.
- * If the type use this value then the size of the array is stored in the object
+ * If the type uses this value, then the size of the array is stored in the object
  * at runtime.
  */
 #define SPAN_DYNAMIC_EXTENT -1
 
 /**
- * Non owning view to a sequence of contiguous elements.
+ * Nonowning view to a sequence of contiguous elements.
  *
  * Spans encapsulate a pointer to a sequence of contiguous elements and its size
  * into a single object. Span can replace the traditional pair of pointer and
- * size arguments passed as array definition in function calls.
+ * size arguments passed as array definitions in function calls.
  *
  * @par Operations
  *
  * Span objects can be copied and assigned like regular value types with the help
- * of copy constructor and copy assignment (=) operator.
+ * of copy constructor and copy assignment (=) operators.
  *
- * Elements of the object can be retrieved with the subscript ([]) operator. The
- * pointer to the first element of the sequence viewed can be accessed with data()
- * while the function size() returns the number of elements in the sequence and
- * empty() informs if the there is any element in the sequence.
+ * You can retrieve elements of the object with the subscript ([]) operator. You can access the
+ * pointer to the first element of the sequence viewed with data().
+ * The function size() returns the number of elements in the sequence, and
+ * empty() informs whether there is any element in the sequence.
  *
- * Span can be sliced from the beginning of the sequence (first()), from the end
+ * You can splice Span from the beginning of the sequence (first()), from the end
```

Comment:
_slice_ is more commonly used in CS in that context.

Reply:
That was a typo on my part. My apologies.